src/styles/Extn-NavigationAndHistory.less

```diff
@@ -236,4 +236,31 @@
 #mrof-container .footer {
   background-color: #2c2c2c;
   border-top: 1px solid #343434;
+}
+
+#mrof-list li {
```

Comment:
nest this css within the recent files panel css div/class to prevent polluting the global css styles